app/assets/stylesheets/application.css

```diff
@@ -16,7 +16,7 @@
  *= require_self
```

Comment:
I know this isn't a thing you did, but `self` and `tree` should really be at the end of the list, right?